awesome_gallery/static/src/gallery_view.js

```diff
@@ -0,0 +1,3 @@
+/** @odoo-module */
+
+// TODO: Begin here!
```

Comment:
if this file is not used then we need to remove it

Reply:
Solved, removed